pkg/source/traderjoev20/constant.go

```diff
@@ -0,0 +1,14 @@
+package traderjoev20
+
+const (
+	DexTypeTraderJoeV20 = "traderjoev20"
```

Comment:
um the convention (for new dex at least) is to use dash
```suggestion
	DexTypeTraderJoeV20 = "traderjoe-v20"
```

Reply:
renamed